e2e/_suites/kubernetes-autodiscover/features/beats.feature

```diff
@@ -0,0 +1,52 @@
+@beats
```

Comment:
Let's add a tag that selects the entire test suite directly. I would rename the base dir to `beats-kubernetes`, acting as a namespace for any kubernetes related tests.
```suggestion
@autodiscover
```

Reply:
Ok, I will review the organization of this. In principle I only wanted to test kubernetes autodiscover in this suite (and not other kubernetes features as metricbeat modules), with beats by now, but possibly with agent too in the near future.

Regarding the feature files, if I expect to have different scenarios for metricbeat, filebeat, hearbeat and agent, would they better go in the same feature file, or would it be prefered to have a different feature file and different tags for each one of them?

Reply:
I'd separate them by feature file, unless they are totally related. OTOH, we can start simple with one feature file, and then split it if/when needed

Reply:
Separated by feature files: 6954b9c26136fa492d0d87eda4d6943ddcf878a9, one for each beat. I have also rephrased the titles of the scenarios to tell more about what it tests, and not about how it does it.